web_widget_mail_send_odoo/static/src/js/web_widget_mail_send_odoo.js

```diff
@@ -104,4 +103,67 @@ openerp.web_widget_mail_send_odoo = function(instance) {
 
     instance.web.form.widgets.add('email_mailto', 'instance.web.form.FieldEmail')
     instance.web.form.widgets.add('email', 'instance.web.form.FieldEmailIntern')
+
+    instance.web_kanban.FieldEmailInternKanban = instance.web_kanban.AbstractField.extend({
+        init: function(parent, field, $node) {
+            this.parent = parent;
```

Comment:
you don't need this. the actual parent is set in https://github.com/OCA/OCB/blob/8.0/addons/web/static/src/js/openerpframework.js#L569. And what you want to keep here is the reference to `this.field_manager`, which another super does already: https://github.com/OCA/OCB/blob/8.0/addons/web/static/src/js/view_form.js#L1815